tests/wpt/mozilla/tests/mozilla/canvas/text-metrics-for-unpaintable-crash.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<!-- https://github.com/servo/servo/issues/36845 -->
+<canvas id="canvas" height="0"></canvas>
+<script>
+  canvas.getContext("2d").measureText("1");
+</script>
```

Comment:
I think this test probably be a shared WPT test in `tests/wpt/tests/html/canvas` since it doesn't rely on anything Servo-specific and it could be quite useful for the rest of the community.

Reply:
Ok